docs/source/getting-started/install.rst

```diff
@@ -1,14 +1,18 @@
 Comprehensive Installation
 ==========================
 
-There are 3 main ways you can install ScanCode.
+There are 4 main ways you can install ScanCode.
 
 - :ref:`app_install`
 
     The recommended method for installing ScanCode is Downloading the latest release as an
     application and then configure and use directly. This is easy because no knowledge of pip/git
     or other developer tools is necessary.
 
+- :ref:`docker_install`
+
+    An alternative to installing the latest Scancode Toolkit release natively is to build a Docker image from the included Dockerfile. This is easy because the only prerequisite is a working Docker install.
```

Comment:
Do you mind avoiding having too lines that are too long? 90'ish characters should be about right.